test/tests/functional/pbs_que_resc_usage.py

```diff
@@ -50,6 +50,19 @@ def setUp(self):
                             'resources_available.ncpus': 10},
                             self.mom.shortname)
 
+    def create_custom_resc(self):
+        """
+        function to create the resource named as "foo"
+        """
+        self.server.manager(MGR_CMD_CREATE, RSC, {
+                            'type': 'long', 'flag': 'q'}, id='foo')
+        self.scheduler.add_resource('foo')
+        self.server.manager(MGR_CMD_SET, QUEUE, {
+                            'resources_available.foo': 6}, id='workq')
+        q_status = self.server.status(QUEUE, id='workq')
```

Comment:
I'm not sure you really need to do this.  If the manager() failed, it would raise an exception.